website/docs/modules/sources.html.markdown

```diff
@@ -44,24 +44,7 @@ Updates for file paths are automatic: when "downloading" the module using the [g
 
 ## Terraform Registry
 
-The [Terraform Registry](https://registry.terraform.io) is an index of modules
-written by the Terraform community.
-The Terraform Registry is the easiest
-way to get started with Terraform and to find modules to start with.
-The registry is integrated directly into Terraform:
-
-```hcl
-module "consul" {
-  source = "hashicorp/consul/aws"
-}
-```
-
-The above example would use the
-[Consul module for AWS](https://registry.terraform.io/modules/hashicorp/consul/aws)
-from the public registry.
-
-You can learn more about the registry at the
-[Terraform Registry documentation section](/docs/registry/index.html).
+See the [Terraform Registry documentation](/docs/registry/modules/use.html#using-modules).
```

Comment:
I'd like to still retain the content here, even though it is indeed a bit redundant with what you linked to, as an example to help users who are just skimming this page to understand the meaning of some code they're looking at. If a user sees `source = "hashicorp/consul/aws"` in some existing config then they may come here to learn what kind of module reference that is.

Reply:
Done!